core/common/src/IModel.ts

```diff
@@ -135,6 +135,15 @@ export interface IModelEncryptionProps {
   readonly password?: string;
 }
 
+/**
+ * Sqlite options.
+ * @public
+ */
+export interface OpenSqliteArgs {
+  /** For more information https://www.sqlite.org/c3ref/busy_timeout.html. */
```

Comment:
Prefer to document what this does rather than redirecting.
At a minimum, document that the units are milliseconds.